nebula_ros/src/hesai/hesai_decoder_ros_wrapper.cpp

```diff
@@ -98,6 +97,9 @@ void HesaiDriverRosWrapper::ReceiveScanMsgCallback(
       rclcpp::Time(SecondsToChronoNanoSeconds(std::get<1>(pointcloud_ts)).count());
     PublishCloud(std::move(ros_pc_msg_ptr), aw_points_ex_pub_);
   }
+
+  auto runtime = std::chrono::high_resolution_clock::now() - t_start;
+  RCLCPP_INFO(get_logger(), "AAA {'d_total': %lu, 'n_out': %lu} BBB", runtime.count(), pointcloud->size());
```

Comment:
Can you please convert this to a DEBUG_INFO?

Reply:
Done. This necessitated a `debug_logging` parameter for the `nebula_launch.py` file to actually enable debug output.